aioshelly/block_device/device.py

```diff
@@ -112,6 +112,11 @@ def ip_address(self) -> str:
         """Device ip address."""
         return self.options.ip_address
 
+    @property
+    def port(self) -> int:
+        """Device ip address."""
+        return self.options.port
```

Comment:
Why do you add the `port` property for gen1 devices if the change only applies to gen2+ devices?

Reply:
Indeed, a leftover of the first attempt to add also gen1 support but as we know from Shelly it won't be available.

Reply:
Reverted all changes for `block_device`